modules/zivid/hand_eye.py

```diff
@@ -169,7 +169,7 @@ def hand_eye_transform(self):
             A 4x4 numpy matrix
 
         """
-        return self.__impl.handEyeTransform()
+        return self.__impl.hand_eyeTransform()
```

Comment:
Functions should be snake_case. (Not camel_snakeCase :))

Reply:
Fixed. This change was not intended. This was the result of a search-replace that was not case-sensitive